app/views/partials/inputDropdown.scala.html

```diff
@@ -7,7 +7,8 @@
 selectedOption: Option[InputNameAndValue],
 multiValue: Boolean=false,
 attributes: Map[Symbol, Any],
-fieldErrors: List[String] = Nil
+fieldErrors: List[String] = Nil,
+defaultOptionText: String = "Please choose...",
 )(implicit messages: Messages)
 
```

Comment:
Not due to your changes but noticed there was some padding between the drop down and the buttons compared to the prototype. We can fix this by changing line 21 from
 `<div class="govuk-form-group govuk-!-margin-bottom-9 @{if(fieldErrors.nonEmpty) "govuk-form-group--error" else ""}">`
to
`<div class="govuk-form-group @{if(fieldErrors.nonEmpty) "govuk-form-group--error" else ""}">`